cli/tools/run/hot_reload/json_types.rs

```diff
@@ -0,0 +1,53 @@
+use deno_core::serde_json::Value;
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct RpcNotification {
+  pub method: String,
+  pub params: Value,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct SetScriptSourceReturnObject {
+  pub status: Status,
+}
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ScriptParsed {
+  pub script_id: String,
+  pub url: String,
+}
```

Comment:
I believe some of these structs are coming from `Chrome Devtools Protocol` - if so I will move them to `cli/tools/repl/cdp.rs`

Reply:
yeah these look a lot like messages from CDP